src/app/services/image-resizer.service.ts

```diff
@@ -17,4 +17,11 @@ export class ImageResizerService {
        );
   }
 
+  disabledButtonColor(resizing: boolean): string {
```

Comment:
I would probably just say remove this functionality all together. a disabled button should be handled globally via the theme styles for continuity.